src/components/metadata/field.js

```diff
@@ -87,7 +100,7 @@ class MetadataField extends React.PureComponent {
         className={cx(classes)}
         onContextMenu={this.handleContextMenu}>
         <label title={details.join('\n\n')}>{label}</label>
-        <div className="value" onClick={this.handleClick}>
+        {this.connect(<div className="value" onClick={this.handleClick}>
```

Comment:
We could introduce a `MetadataValue` component in a separate file to keep the dnd logic away from the `MetadataField` component.